### PR TITLE
fixed link to agronomic metadata tutorial

### PR DESCRIPTION
**Pull request by kimberlyh66**

### CODE REVIEW

File: traits/01-web-access.Rmd [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+6---
```
Comment (dlebauer) [inline traits/01-web-access.Rmd]: did you intend to put this six here?

Or should it still be 

```suggestion
---
```

Comment (kimberlyh66) [reply traits/01-web-access.Rmd]: No I did not intend to place the 6 there -- I will remove it.

File: traits/02-betydb-api-access.Rmd [hunk 2]
```diff
@@ -1,107 +1,118 @@
...
+What do you see? Do you think that this is all of the records? What happens if you add `&limit=none`? 
+
+
+#### Accessing data using the Shell
+
+Type the following command into a bash shell (the -O option names the output file): 
+
+```sh
+wget -O sorghum.json
+   "https://terraref.ncsa.illinois.edu/bety/api/beta/species?genus=Sorghum&key=9999999999999999999999999999999999999999"
```
Comment (dlebauer) [inline traits/02-betydb-api-access.Rmd]: put the `\` in (`\\` doesn't work), as well as the comment.

please replace all `wget -O` with `curl -o` to make it more platform independent (e.g. work on macs where wget isn't available by default)

File: traits/02-betydb-api-access.Rmd [hunk 3]
```diff
@@ -1,107 +1,118 @@
...
+```
+
+If you want to write the query without exposing the key in plain text, you can construct it like this:
+
+```sh
+wget -O sorghum.json
+    "https://terraref.ncsa.illinois.edu/bety/api/beta/species?genus=Sorghum&key=`cat ~/.betykey_public`"
+```
+
+> Do you know what `cat ~/.betykey_public`. Run the command and see what it outputs in your shell.
```
Comment (dlebauer) [inline traits/02-betydb-api-access.Rmd]: you can delete this stuff

File: traits/04-danforth-indoor-phenotyping-facility.Rmd [hunk 4]
```diff
@@ -26,7 +27,7 @@ library(traits)
 Unlike the first two tutorials, now we will be querying real data from the public TERRA REF database. So we will use a new URL, https://terraref.ncsa.illinois.edu/bety/, and we will need to use our own private key.
 
 ```{r terraref-connect-options}
-options(betydb_key = readLines('~/.betykey', warn = FALSE),
+options(betydb_key = readLines('.betykey', warn = FALSE),
```
Comment (dlebauer) [inline traits/04-danforth-indoor-phenotyping-facility.Rmd]:
```suggestion
options(betydb_key = readLines('.betykey'),
```

**Pull request merged**